src/sentry/static/sentry/app/styles/spacingScale.jsx

```diff
@@ -0,0 +1,28 @@
+export default size => {
+  if (size > 9) return (size - 9) * 100;
+
+  switch (size) {
+    case 0:
+      return 0;
+    case 1:
+      return 2;
+    case 2:
+      return 4;
+    case 3:
+      return 6;
+    case 4:
+      return 10;
+    case 5:
+      return 16;
+    case 6:
+      return 20;
+    case 7:
+      return 24;
+    case 8:
+      return 36;
+    case 9:
```

Comment:
I'm concerned this is too many options.

Reply:
that's fine. i went to 10 mostly because it seems like a nice place to stop.

two questions:

1. where should we stop? 6? 5?
2. what should we do if a developer adds something over the limit? right now i just multiply it by ten but we could also throw an error.